### PR TITLE
Updated CLIENT LIST and KILL doc to include laddr

### PR DESCRIPTION
**Pull request by yoav-steinberg**

### CODE REVIEW

**Review by itamarhaber — COMMENTED** [review]
Thank you @yoav-steinberg - please document this under the `@history` section in the doc

File: commands/client-kill.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ With the new form it is possible to kill clients by different attributes
 instead of killing just by address. The following filters are available:
 
 * `CLIENT KILL ADDR ip:port`. This is exactly the same as the old three-arguments behavior.
+* `CLIENT KILL LADDR ip:port`. Kill all clients connected to specified local (bind) address.
```
Comment (oranagra) [inline commands/client-kill.md:16]: need to somewhere mention which version of redis ( >= 6.2 )? this command is supported.
probably do the same for the field.

Comment (itamarhaber) [reply commands/client-kill.md:16]: Duplicate of https://github.com/redis/redis-doc/pull/1424#pullrequestreview-519662023

**Comment by yoav-steinberg** [comment]
> Thank you @yoav-steinberg - please document this under the `@history` section in the doc

@itamarhaber, can you point me to where that is?

**Comment by oranagra** [comment]
Some examples
https://redis.io/commands/client-list (ID) 
https://redis.io/commands/client-kill  (ID) 
https://redis.io/commands/zadd (history at the bottom)

File: commands/client-kill.md (hunk 1 above)
Comment (yoav-steinberg) [reply commands/client-kill.md:15]: Done.

**Comment by yoav-steinberg** [comment]
> Some examples...
 done. please review.

File: commands/client-kill.md [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
     CLIENT KILL <filter> <value> ... ... <filter> <value>
 
 With the new form it is possible to kill clients by different attributes
 instead of killing just by address. The following filters are available:
 
 * `CLIENT KILL ADDR ip:port`. This is exactly the same as the old three-arguments behavior.
-* `CLIENT KILL ID client-id`. Allows to kill a client by its unique `ID` field, which was introduced in the `CLIENT LIST` command starting from Redis 2.8.12.
-* `CLIENT KILL TYPE type`, where *type* is one of `normal`, `master`, `slave` and `pubsub` (the `master` type is available from v3.2). This closes the connections of **all the clients** in the specified class. Note that clients blocked into the `MONITOR` command are considered to belong to the `normal` class.
+* `CLIENT KILL LADDR ip:port`. Kill all clients connected to specified local (bind) address.
+* `CLIENT KILL ID client-id`. Allows to kill a client by its unique `ID` field.
```
Comment (oranagra) [inline commands/client-kill.md:16]: i see the old text was mentioning CLIENT LIST, maybe we wanna keep that? or refer to cLIENT ID too?

File: commands/client-list.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
-The `CLIENT LIST` command returns information and statistics about the client
+The `CLIENT LIST [TYPE type]` command returns information and statistics about the client
```
Comment (oranagra) [inline commands/client-list.md:1]: @itamarhaber will that change cause the link to the client list page to stop working?

File: commands/client-kill.md (hunk 2 above)
Comment (yoav-steinberg) [reply commands/client-kill.md]: Fixed.

File: commands/client-list.md (hunk 3 above)
Comment (itamarhaber) [reply commands/client-list.md:1]: Yep, but that's not terrible as the user is already on that page. However, I don't think the syntax needs to be repeated here, so this should be reverted.

**Comment by itamarhaber** [comment]
@yoav-steinberg thanks for adding history, that's more than perfect (and my apologies for not providing clearer instructions).

File: commands/client-list.md (hunk 3 above)
Comment (yoav-steinberg) [reply commands/client-list.md]: ok

**Review by oranagra — APPROVED** [review]

**Pull request merged**